app/Main.hs

```diff
@@ -30,6 +34,7 @@ commitInfo = case gitInfo of
 
 data CLI
   = Run AppOpts
+  | ListKeybinding Bool KeybindingPrint
```

Comment:
Add haddock for this `Bool` parameter?  Or alternatively define a self-documenting Bool-isomorphic type

Reply:
I can not come up with a good name for the type "initialize the INI file or not," so doc it is. 😄